configuration/wireless/wireless-cli.md

```diff
@@ -34,6 +47,8 @@ network={
     psk="testingPassword"
 }
 ```
+
+If you are using `wpa_passphrase` you can redirect its output to your configuration file by calling `wpa_passphrase "testing" >> /etc/wpa_supplicant/wpa_supplicant.conf`. Note that this requires you to change to `root` (by executing `sudo su`) or find another way to redirect the output since the file we write to can be altered only by a user with administrative privileges. Last but not least make sure you use `>>` (which is used to append text to an existing file) since `>` will erase all contents and **then** append the output to the specified file.
```

Comment:
Don't forget to purge the wpa_passphrase command from the bash_history or you will still have the psk saved in clear text.